packages/DataGrid/stories/DataGrid.mdx

```diff
@@ -0,0 +1,372 @@
+</DataGrid>
+
+Note: Any information can be stored in the object.
+
+### Custom cell content
+
+The DataGrid component provides a render prop for `cell` so that the consumer can change what the cell displays.
+
+For example, if we wanted to include a 👋 emoji, we would do the following.
+
+```js
+const data = [
+  /* column */ /* colum */
+  { myColum1: 1, myColum2: "hi" }, // row 1
+  { myColum1: 2, myColum2: "hello" }, // row 2
+  { myColum1: 3, myColum2: "hola" }, // row 3
+];
+
+<DataGrid data={data}>
+  <DataGrid.ColumnDefinition header="Column 1" cell="myColum1" />
+  <DataGrid.ColumnDefinition
+    header="Column 2"
+    cell={({ row }) => (row.myColum2 === "hi" ? `👋 ${myColum2}` : myColum2)}
+    cellA11yText={() => "custom a11y text"}
+  />
+</DataGrid>;
+```
+
+#### Result
+
+<DataGrid
+  data={[{ myColum1: 1, myColum2: "hi" }, { myColum1: 2, myColum2: "hello" }, { myColum1: 3, myColum2: "hola" }]}
+>
+  <DataGrid.ColumnDefinition header="Column 1" cell="myColum1" />
+  <DataGrid.ColumnDefinition
+    header="Column 2"
+    cell={({ row }) => (row.myColum2 === "hi" ? `👋 ${row.myColum2}` : row.myColum2)}
+    cellA11yText={() => "custom a11y text"}
+  />
+</DataGrid>
+
+<br />
+
+<Toast hasCloseButton={false} kind="warning">
+  If a custom cell is used, you must also include the cellA11yText prop.
+</Toast>
+
+### Styling cells
+
+The `cellProps` prop is used to add custom styling to the cells.
+
+In the following example, when the cell has the letter 'o', then we add a blue background to the cell.
+
+```js
+
+const style = {
+  background: `${row.myColum2.includes("o") ? '#cce5fd' : "transparent"}`
+}
+
+<DataGrid data={data}>
+  <DataGrid.ColumnDefinition header="Column 1" cell="myColum1" />
+  <DataGrid.ColumnDefinition header="Column 2"
+    cell={
+      ({row}) => (row.myColum2 === "hi") ? `👋 ${row.myColum2}` : row.myColum2
+    }
+    cellProps={({ row }) => ({ style })
+  }
+  />
+</DataGrid>
+```
+
+#### Result
+
+<DataGrid
+  data={[{ myColum1: 1, myColum2: "hi" }, { myColum1: 2, myColum2: "hello" }, { myColum1: 3, myColum2: "hola" }]}
+>
+  <DataGrid.ColumnDefinition header="Column 1" cell="myColum1" />
+  <DataGrid.ColumnDefinition
+    header="Column 2"
+    cell={({ row }) => (row.myColum2 === "hi" ? `👋 ${row.myColum2}` : row.myColum2)}
+    cellProps={({ row }) => ({
+      style: {
+        background: `${row.myColum2.includes("o") ? "#cce5fd" : "transparent"}`,
+      },
+    })}
+  />
+</DataGrid>
+
+<br />
+<Toast kind="info" hasCloseButton={false}> Tip: `cell` and `cellProps` are functions that could be memoized</Toast>
+
+### Sticky headers
+
+By default all our header are sticky and mandatory, so you will have this feature enable, without doing anything, enjoy it 😄.
+
+### Sticky columns
+
+1. Add the `isSticky` prop to the `<DataGrid.ColumnDefinition/>`
+2. Make sure the `isSticky` columns are grouped together before the columns that are not sticky. This prevents issues with keyboard navigation.
```

Comment:
```suggestion
2. Make sure the `isSticky` columns are grouped together before the columns that are not sticky. This will prevent issues with keyboard navigation.
```